### PR TITLE
Rc67.2 Provide a toggle switch for TAA in the menu developer/render to opt-out

### PR DESCRIPTION
**Pull request by samcake**
Adding  toggle switch for TAA in the menu developer/render to opt-out and go back to FXAA (before TAA was introduced)
TAA is ON by default.
image

TEST PLAN
- verify that the new menu item (in developer/render)appears and behave as expected between re runs
  - ON by default after fresh install
  - Only accessible after enabling the developer ui
  - If toggled on / off during a session, quit, restart and it should still be in the same state as when quit
 
- Verify that the Antialiasing used in the main view used to render the image reflects the menu item value
   - Run the application in desktop mode fullscreen in a HD display (1k x 2k)
   - Going into "asimov" and facing the palm tree, observing them moving 
       - It should be obvious that with TAA ON, they look smooth, eventually a bit blurry as they move
       - With TAA OFF, they look very much Aliased

    - Repeat the same observation with HMD

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9859/HighFidelity-Beta-PR13185-ade1d6c0567abed81bff1802a491cd2400adc9ab.apk

File: libraries/render-utils/src/AntialiasingEffect.h [hunk 1]
```diff
@@ -95,7 +95,7 @@ class AntialiasingConfig : public render::Job::Config {
 
     Q_PROPERTY(bool debug MEMBER debug NOTIFY dirty)
     Q_PROPERTY(float debugX MEMBER debugX NOTIFY dirty)
-    Q_PROPERTY(float debugFXAAX MEMBER debugFXAAX NOTIFY dirty)
+    Q_PROPERTY(bool fxaaOnOff READ debugFXAA WRITE setDebugFXAA NOTIFY dirty)
```
Comment (SamGondelman) [inline libraries/render-utils/src/AntialiasingEffect.h]: We should call this something clearer, like just fxaaOn.

Comment (SamGondelman) [reply libraries/render-utils/src/AntialiasingEffect.h]: Similarly, "debugFXAAX" is confusing.  It looks like fxaaOnOff = true -> debugFXAAX = 0.0f.  Can we make them match?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9859/HighFidelity-Beta-PR13185-ade1d6c0567abed81bff1802a491cd2400adc9ab.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9859/HighFidelity-Beta-Interface-PR13185-ade1d6c0567abed81bff1802a491cd2400adc9ab.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9859/HighFidelity-Beta-PR13185-ade1d6c0567abed81bff1802a491cd2400adc9ab.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9859/HighFidelity-Beta-Interface-PR13185-ade1d6c0567abed81bff1802a491cd2400adc9ab.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Pull request merged**